data/hai/unfettered jobs.txt

```diff
@@ -324,13 +324,14 @@ mission "Unfettered Small Medicals Rush"
 		attributes "spaceport"
 	destination "Darkcloak"
 	to offer
-		random < 30 + "Unfettered Jump Drive 1: offered" * 30
+		random < 25 + "Unfettered Jump Drive 1: offered" * 25
+		not "flagship planet: Darkcloak"
 	on visit
 		dialog phrase "generic missing stopover or cargo"
 	on stopover
 		dialog phrase "unfettered medicals pickup dialog"
 	on complete
-		payment 8000 3300
+		payment 8000 3000
```

Comment:
The default payment multiplier is 150. A few rare jobs reach up in the 200s. Very rarely might you see something in the 500s (for some pirate jobs, for example). 3300 is totally out of band, and 3000 is hardly a nerf.

Reply:
the tributes go up to 2800, this is high level of money on purpose, as is reflected by the super tight deadline - no other jobs have a 0 tolerance for error as far as I know, and this is a very small amount of cargo too

then there's also the need to go deep in unfettered space with bribes that cost a bunch, and you may even die if you get this pre-JD
post JD it's part of why it's worth it to sell for so little; you get access to good jobs but it limits the size of your fleet

this job is one of the only ones that forces you to keep a tiny fleet, too, as far as I'm aware

Reply:
if we need to do smth further I'd prefer to not just bring the money down - that's the easiest solution, sure, but often not the interesting one
we could bring the cargo even lower, for instance